models/post_struct.go

```diff
@@ -98,7 +98,6 @@ type PostListingChildData struct {
 	LinkFlairBackgroundColor   string             `json:"link_flair_background_color"`
 	Id                         string             `json:"id"`
 	IsRobotIndexable           bool               `json:"is_robot_indexable"`
-	ReportReasons              string             `json:"report_reasons"`
```

Comment:
Thanks for the comment regarding this deprecated feature! I agree that it should not be used, however, I am not sure how many programs out there might inadvertently use it. In spirits of not breaking compatibility, let's just leave it there. (we can remove it and bump mira to a new major version)